src/utils/outputs/swiftui/AdaptiveColors+Descripitve.swift

```diff
@@ -0,0 +1,233 @@
+//
```

Comment:
We don't have any Licence information in other files. Should we remove it here @dennispost?